common/views/components/FeaturedText/README.md

```diff
@@ -0,0 +1,5 @@
+## Purpose
+
+Used to emphasise a paragraph of text. The first para on an exhibition page, for example.
```

Comment:
Should examples maybe be a section? Bikeshedding for a bigger cardy convo though.

Reply:
Maybe, yeah. I've been trying to leave a link to edit the readmes on GitHub to make it easier for non-devs to edit. Obviously forgot to do so here. Will update.